mavenRepo/org/robobinding/aspectj-plugin/0.8.4/aspectj-plugin-0.8.4.jar.md5

```diff
@@ -1 +1 @@
-0d2d30bffed5714a3f623be3ac521e63
```

Comment:
@xudeheng I think that is good enough. People could either choose 0.8.4 plus old Android studio or 0.8.5+ plus Android Studio 2.+.